src/main/xerus/monstercat/api/Covers.kt

```diff
@@ -32,9 +32,16 @@ object Covers {
 		}
 		return file.inputStream()
 	}
-	
+  
 	private fun coverCacheFile(coverUrl: String) =
 		coverCacheDir.resolve(coverUrl.substringAfterLast('/').replaceIllegalFileChars())
+  
+	fun getLargeCoverImage(coverUrl: String, size: Int = 1024): Image =
+			getLargeCover(coverUrl, size).use { Image(it, size.toDouble(), size.toDouble(), false, false) }
+	
+	fun getLargeCover(coverUrl: String, size: Int): InputStream =
+			fetchCover(coverUrl, size).content
```

Comment:
Since we now use the large covers in multiple places, we should consider caching them as well, or?

Reply:
If we use the large covert anywhere else, why not, but this will take massive disk space after a while.

Reply:
That's why we need to automatically prune the cache. There's already such a mechanism for the logs.

Reply:
but that's not urgent

Reply:
Well it's obsolete; I cache the releases now

Reply:
huh?

Reply:
The large covers are cached like the thumbnails now. Marking as resolved